build/Dockerfile

```diff
@@ -79,7 +79,16 @@ RUN apt-get update && \
     && cp /root/.gitconfig /etc/gitconfig
 
 
+# Add bear tool to be able to generate compilation database during build step
+RUN set -ex; \
+    git clone --branch 2.4.4 https://github.com/rizsotto/Bear.git && \
+    cd Bear && \
+    cmake -DCMAKE_INSTALL_PREFIX=/opt/bear . && \
+    make all install
```

Comment:
please remove sources / build dir at the end (no need for Bear sources in the resulting docker image)

Reply:
done